hash-params.ts

```diff
@@ -0,0 +1,28 @@
+export interface HashParams {
+  profileURL?: string
+  title?: string
+}
+
+export default function getHashParams(): HashParams {
```

Comment:
I haven't really been using default exports for anything anywhere, and IIRC, it makes mocking things kind of a pain. Would you might switching this to just a regular exported function?

Reply:
Sure. Not sure about the details of the mocking you're talking about, but at least with the normal commonjs transform, it is possible to mock default exports, I'm pretty sure.